src/main/resources/messages/UiLabelsBundle.properties

```
@@ -10,8 +10,8 @@ editor=Editor
 foldingRangeProvider=Folding Range Provider
 defaultFormatter=Default Formatter
 notebookFormatter=Default Notebook Formatter
-enableCodeLensesComment=Enable CodeScene Code Lenses
-enableAutoRefactorComment=Enable CodeScene ACE. This is currently only available for customers part of the ACE beta program.
+enableCodeLensesComment=Show Code Vision for review diagnostics
```

nit: would be good to keep the title case capitalization here to match the rest

```suggestion
enableCodeLensesComment=Show Code Vision for Review Diagnostics
```